core-bundle/src/Controller/ContentElement/VideoController.php

```diff
@@ -102,11 +109,11 @@ private function getVimeoSourceParameters(ContentModel $model): array
 
         return [
             'provider' => 'vimeo',
-            'video_id' => $videoId = $model->vimeo,
+            'video_id' => $videoId = explode('?', $model->vimeo)[0],
             'options' => $options,
-            'base_url' => $baseUrl = "https://player.vimeo.com/video/$videoId",
+            'base_url' => $baseUrl = "https://player.vimeo.com/video/$model->vimeo",
             'query' => $query,
-            'url' => empty($query) ? $baseUrl : "$baseUrl?$query",
+            'url' => "https://player.vimeo.com/video/$videoId".($query ? "?$query" : ''),
```

Comment:
So now the `base_url` property contains the privacy hash but the `url` property does not. Is this correct?

Reply:
Both contain the hash as the `$query` variable contains the hash too, see line 87.